### PR TITLE
feat(DeploymentServerUI): Proper Monitoring & Terminal

### PR DESCRIPTION
**Pull request by Gozzim**
## Changes Proposed:

- Proper monitoring connection
- Decent Terminal component and styling
- Device Notifications now link to their page
- Various bug fixes & minor changes

## How to Test PRs
1) First fetch the Pull-Request from the main repo, replacing `xx` with the ID of the PR:\
`git fetch git@github.com:eclipsesource/cdtcloud-deploymentserver.git pull/xx/head:pr_xx`
2) Checkout the Pull-Request, again replacing `xx` with the PR-ID:\
`git checkout pr_xx`
3) Perform the necessary tests for the PR

### CODE REVIEW

File: packages/deployment-server-ui/src/components/Notification.tsx [hunk 1]
```diff
@@ -85,15 +85,18 @@ export const deviceEvent = async (resp: ServerMessage): Promise<void> => {
     }
 
     const deviceTypeName = await typeIdToName(device.deviceTypeId)
+    const title = (<a href={'/devices'} style={{ color: 'black' }}>{`Device ${titleEvent}`}</a>)
     const deviceMessage = (
-      <>
+      <a href={'/devices'} style={{ color: 'black' }}>
```
Comment (WoH) [inline packages/deployment-server-ui/src/components/Notification.tsx:90]: Link

File: packages/deployment-server-ui/src/components/Notification.tsx [hunk 2]
```diff
@@ -85,15 +85,18 @@ export const deviceEvent = async (resp: ServerMessage): Promise<void> => {
...
         Id: {device.id}
         <br/>
         Connector: {device.connectorId}
         <br/>
-        Type: {deviceTypeName}
-      </>
+        <a href={`types/${device.deviceTypeId}`} style={{ color: 'black' }}>
```
Comment (WoH) [inline packages/deployment-server-ui/src/components/Notification.tsx:95]: Link

Comment (Gozzim) [reply packages/deployment-server-ui/src/components/Notification.tsx:95]: Not possible due to the function not being a ReactComponent.

File: packages/deployment-server-ui/src/components/Notification.tsx (hunk 1 above)
Comment (Gozzim) [reply packages/deployment-server-ui/src/components/Notification.tsx:90]: Not possible due to the function not being a ReactComponent.

Comment (WoH) [reply packages/deployment-server-ui/src/components/Notification.tsx:90]: Please add a comment then, as we both agree that there are better ways to approach this (i.e. onClick).

**Review by WoH — APPROVED** [review]

**Pull request merged**